### DISCUSSION TITLE
Glances displays as WHITE Screen inside any browser window

### DISCUSSION THREAD

**Discussion by 1artist** [nicolargo/glances#2483] [General]
My Glances installation has been very difficult. I followed the steps in the YouTube link below:
[https://www.youtube.com/watch?v=4OOlQCJXKh8](url)

I think the problem is with bottle. Below are two different ways I installed bottle.  I still get a white screen inside the my Edge browser and chrome browser and Librewolf browser.  

john@myserver:~# sudo pip install bottle
Requirement already satisfied: bottle in /usr/lib/python3/dist-packages (0.12.19)

john@myserver:~# sudo pip3 install bottle
Requirement already satisfied: bottle in /usr/lib/python3/dist-packages (0.12.19)

**Describe the bug**
Cannot see Glances in any browser. All I see is a white Screen in any browser window. However, I can see Glances inside my terminal.
When I type the command below I get a [Errno 98] error shown below:
john@myserver:~$ glances -w
Glances Web User Interface started on http://0.0.0.0:61208/
Error: Can not ran Glances Web server ([Errno 98] Address already in use)
 

**To Reproduce**
1. Entered this command: glances -w
2. Get a [Errno 98]  error.

**Expected behavior**
I expect to see glances inside any browser window and all I see is a WHITE screen inside any browser.

 - Operating System - Ubuntu Server 22.04 
 - Glances & psutil versions: Glances v3.4.0.3 with PsUtil v5.9.5
 - How do you install Glances (Pypi package, script, package manager, source): `To be completed`

```
My Install Steps below:
1. apt-get install glances -y
2. glances --version
3. systemctl status glances
4. glances
5. nano /etc/glances/glances.conf
6. nano /usr/lib/systemd/system/glancesweb.service

[Unit]
Description = Glances in Web Server Mode
After = network.target

[Service]
ExecStart = /usr/bin/glances -w --bind 192.xxx.x.xxx --port 61208 -t 5
ALSO TRIED THIS CODE:   ExecStart = /usr/bin/glances  -w  -t  5

[Install]
WantedBy = multi-user.target

7. systemctl daemon-reload
8. systemctl start glancesweb
9. systemctl enable glancesweb
10. systemctl status glancesweb

.To fix a different error:
11. apt install python3-pip
12. pip install bottle

ss -antpl | grep glances

```
In closing below is the result I get when I run a "ss -antpl | grep glances" commanad:
john@myserver:~$ sudo ss -antpl | grep glances
LISTEN 0      5      192.xxx.x.xxx:61208      0.0.0.0:*    users:(("glances",pid=831,fd=18))                                                                           
LISTEN 0      5          127.0.0.1:61209      0.0.0.0:*    users:(("glances",pid=830,fd=4))   
```

**Comment by nicolargo** [comment]
Hi @1artist 

mixing apt installation and pip installation do not work.

Please uninstall Glances from the Ubuntu package manager and follow the PIp install instruction https://github.com/nicolargo/glances#installation
